scalameta/tokens/shared/src/main/scala/scala/meta/tokens/Token.scala

```diff
@@ -132,12 +154,12 @@ object Token {
   }
 
   // Trivia
-  @fixed(" ") class Space extends Token
-  @fixed("\t") class Tab extends Token
-  @fixed("\r") class CR extends Token
-  @fixed("\n") class LF extends Token
-  @fixed("\f") class FF extends Token
-  @freeform("comment") class Comment(value: String) extends Token
+  @fixed(" ") class Space extends HSpace
+  @fixed("\t") class Tab extends HSpace
+  @fixed("\r") class CR extends AtEOL
```

Comment:
Why `AtEOL` ? Not sure what does this category mean?

Reply:
see comment above. `CR` and `LFLF`, which are part of it, are not _really_ EOL themselves, but they are indicators that an end of line is nearby :)